internal/bminventory/inventory.go

```diff
@@ -1254,6 +1255,76 @@ func (b *bareMetalInventory) InstallClusterInternal(ctx context.Context, params
 	return cluster, nil
 }
 
+func (b *bareMetalInventory) InstallSingleDay2HostInternal(ctx context.Context, clusterId strfmt.UUID, hostId strfmt.UUID) error {
```

Comment:
I guess you can refactor the current `InstallSingleDay2Host` implementation to use this, no?

Reply:
There are several transactions and loops in `InstallHosts` and it makes it hard to refactor the function to work on a single host.
Can you look at `InstallHosts` and tell me if it worth the effort?